cmd/init.go

```diff
@@ -200,6 +206,33 @@ func validateBlockchainProvider(providerString, nodeString string) error {
 	return nil
 }
 
+func validateQuorumConsensus(consensusString string) error {
+	v, err := fftypes.FFEnumParseString(context.Background(), types.QuorumConsensus, consensusString)
+	if err != nil {
+		return nil
+	}
+
+	if v != types.QuorumConsensusClique {
+		return errors.New("support for raft/ibft/qbft will come in future")
```

Comment:
```suggestion
		return errors.New("consensus algorithms such as raft/ibft/qbft for quorum not supported")
```

Reply:
Raised the following issues:
1. https://github.com/hyperledger/firefly-cli/issues/312
2. https://github.com/hyperledger/firefly-cli/issues/311